README.md

```diff
@@ -1,117 +1,216 @@
+The string is interpreted as a regular expression, and only tests whose description (including any group descriptions) match that regular expression will be run. You can also use the -N flag to run tests whose names contain a plain-text string.
 
-## Running the tests
 
-pub run test
+### Test Descriptions
 
-### Tests Descriptions
 
-Test Name:
 Test #1: Loading the site returns valid content with 200 response code
+
 Test #2: Check base url navigates to /home
+
 Test #3: Check there is no error messages in the console
+
 Test #4: Check at least 4 elements in the "Szolgaltatasok" menu
+
 Test #5: Check "Szolgaltatasok" menu contains an instance of "Videotar
+
 Test #6: Check login redirects to login page
+
 Test #7: Register a new user
+
 Test #8: Login with a new user
+
 Test #9: Login with an old user
+
 Test #10 User is not logged in by default
+
 Test #11: Check there is a profile IMG after logging in
+
 Test #12: Check you see at least 8 elements under Szolg�ltat�saink menu
+
 Test #13: After login, there is at least 8 elements in Szolg�ltat�saink menu
+
 Test #14: Check there is at least 6 video elements under /workouts
+
 Test #15: Check at least 5 elements (rows) under "Hol �rhet? el?"
+
 Test #16: Check at least 1 "Vide�t�r" under "Hol �rhet? el?
+
 Test #17: Box on left side of page has data not just labels
+
 Test #18: Check you see a bunch of videos showing up
+
 Test #19: Try to play the video, check if the popup shows up on page
-Test #20: Check if the filtering options work
-Test #21: After login, Check user shows on all Highscore lists
-Test #22 (�r�kranglista): Scores Page Birth-List Tab has data and includes users name
-Test #23 (30 napos): Scores Page 30-Days Tab has data and includes users name
-Test #24 (egy h�napban): Scores Page 1-Month-Ago Tab has data and includes users name
-Test #25: Check clicking on "Megvasarlom" throws no errors | The GWL
-Test #26: Clicking on "Megvasarlom" throws no errors | Superhuman Program
 
-### And coding style tests
+Test #20: Check if the workout type filtering options work
+
+Test #21: Check if the workout time filtering options work
+
+Test #22: Check if the text filtering options work
+
+Test #23: After login, Check user shows on all Highscore lists
 
-Explain what these tests test and why
+Test #24 (�r�kranglista): Scores Page Birth-List Tab has data and includes users name
+
+Test #25 (30 napos): Scores Page 30-Days Tab has data and includes users name
+
+Test #26 (egy h�napban): Scores Page 1-Month-Ago Tab has data and includes users name
+
+Test #27: Check clicking on "Megvasarlom" throws no errors | The GWL
+
+Test #28: Clicking on "Megvasarlom" throws no errors | Superhuman Program
+
+Test #29: Clicking on "Megvasarlom" throws no errors | Dancing Mom
+
+Test #30: Clicking on "Megvasarlom" throws no errors | Joga Mania
+
+
+
+#### Directory And File Structure
+
+Dart has a prescribed directory structure in order to ensure that its tools work
+out of the box.
```

Comment:
It has indeed, but you did not seem to follow it. There is no recommended `libs` directory, just `lib`, but that should contain dart source files, not 3rd party jars.